lib/kubeclient.rb

```diff
@@ -13,6 +13,9 @@ module Kubeclient
   class Client < Common::Client
     attr_reader :api_endpoint
 
+    SERVICE_ACCOUNT_TOKEN_FILE =
```

Comment:
Well OK - it's your client after all... I can just see this leading to lots of wasted time by devs using this as they hunt down typos in the token file name & actually finding the token file name... but I''ll remove it at your request.